packages/legacy/src/client/helpers.ts

```diff
@@ -167,6 +181,14 @@ export const ovirtUrlToHostname = (url: string): string => {
 export const ovirtHostnameToUrl = (hostname: string): string =>
   `https://${hostname}/ovirt-engine/api`;
 
+export const openstackUrlToHostname = (url: string): string => {
+  const match = url.match(/^http:\/\/(.+)\:5000\/v3$/);
```

Comment:
```suggestion
  const match = url.match(/^http[s]?:\/\/(.+)\(:[0-9]+)?/);
```

Reply:
so inside the trusted network it's http and v3 
https://docs.openstack.org/keystone/latest/install/keystone-install-rdo.html
public port defaults to 5000 but is configurable
https://docs.openstack.org/keystone/latest/configuration/config-options.html#eventlet_server.public_port

but from the outside, all is up to the user:
https://docs.openstack.org/keystone/latest/install/keystone-install-rdo.html#configure-the-apache-http-server

so in real life clusters it will be https with changed the path and port